### PR TITLE
Regenerate project files.

### PR DESCRIPTION
**Pull request by nicolasnoble**

### CODE REVIEW

File: vsprojects/grpc_csharp_ext/grpc_csharp_ext.vcxproj [hunk 1]
```diff
@@ -80,6 +80,9 @@
   <PropertyGroup Label="UserMacros" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <TargetName>grpc_csharp_ext</TargetName>
+    <Linkage-grpc_dependencies_zlib>static</Linkage-grpc_dependencies_zlib>
```
Comment (jtattermusch) [inline vsprojects/grpc_csharp_ext/grpc_csharp_ext.vcxproj]: Does this mean we are planning to always link zlib statically and distribute it as a part of gRP?. This also means that we would need to drop the dependency on https://www.nuget.org/packages/grpc.dependencies.zlib/.

**Comment by jtattermusch** [comment]
looking at jenkins, this would break Windows.

**Comment by ctiller** [comment]
I see the same failures at head as soon as anyone generates project files.

**Comment by nathanielmanistaatgoogle** [comment]
@nicolasnoble: would you like @soltanmm and I to hold off on submitting #2697 until this is merged or should we merge that now and it won't add extra work for you?

**Comment by ctiller** [comment]
Can we close this then?

**Comment by nicolasnoble** [comment]
Yes.

**Pull request closed without merging**